js/pro/binance.js

```diff
@@ -1037,6 +1047,15 @@ module.exports = class binance extends binanceRest {
         }
     }
 
+    async resetConnection (client) {
+        const interval = this.safeInteger (this.options, 'resetConnectionInterval', 86100000);
+        const hours = Precise.stringDiv (this.numberToString (interval), '3600000');
+        const error = new ResetConnection ('Resetting connection after ' + hours + ' hours. Binance will close the connection after 24 hours. Catch this error and call call again watch function to reconnect. This can be disabled by setting the option resetConnectionInterval to 0');
+        client.reject (error);
+        client.close ();
```

Comment:
python throws a runtime error suggesting to await this line. However if awaited, binance will throw a both error 1000 and error 1006

Reply:
@pcriadoperez yeah because `close()` is async, do we need to call `close` here?

Reply:
I think so. At the end we need to close the connection, so it can restart on the next watch call.